packages/KSelect/KSelectItem.vue

```diff
@@ -81,6 +81,19 @@ export default {
     .k-select-item-label {
       width: auto;
       line-height: 16px;
+      font-weight: 500;
+      font-size: 14px;
+      padding: 8px;
```

Comment:
fix: can we add the color and bottom margin here (I forgot to include in the ticket)
```suggestion
      color: var(--grey-600);
      font-weight: 500;
      font-size: 14px;
      padding: 8px;
      margin-bottom: 4px;
```

Reply:
Updated